src/style/colors.ts

```diff
@@ -15,7 +15,11 @@ export type DesignToken = {semantic?: [string]; resource_paths?: [string]; toStr
 export type TokensOptions = {primaryColor: string};
 export type GetColorTintOptions = {avoidReverseOnDark?: boolean};
 export type GetColorByHexOptions = {validColors?: string[]};
-
+export type PaletteOptions = {
+  lightnessFix?: boolean;
+  saturationFix?: boolean;
+  darkModeColors?: boolean;
```

Comment:
The options is not very clear and doesn't sound like a boolean option. 
Maybe rename to something like `forDarkModeColors` 
I would also consider adding a short comment description about each option, so it will appear in the typings info

Reply:
Added comments. How about renaming to 'addDarkestColors'?

Reply:
Much better, maybe `addDarkestTints`